source/install/yum.rst

```diff
@@ -18,50 +18,55 @@ of the following distributions:
    :local:
 
 Package Contents
-================
+================================================================================
+
+.. list-table::
+   :widths: 25 75
+
+   * - Package
+     - Contains
+   * - percona-server-mongodb-40
+     - The ``mongo`` shell, import/export tools, other client
+       utilities, server software, default configuration, and init.d scripts.
+   * - percona-server-mongodb-40-server
+     - The :program:`mongod` server, default configuration files, and :dir:`init.d`
+       scripts
+   * - percona-server-mongodb-40-shell
+     - The ``mongo`` shell
+   * - percona-server-mongodb-40-mongos
+     - The ``mongos`` sharded cluster query router
+   * - percona-server-mongodb-40-tools
+     - Mongo tools for high-performance MongoDB fork from Percona
+   * - percona-server-mongodb-40-dbg
+     - Debug symbols for the server
 
-:``Percona-Server-MongoDB-36``:
- Installs the ``mongo`` shell, import/export tools, other client utilities,
- server software, default configuration, and init.d scripts.
-
-:``Percona-Server-MongoDB-36-server``:
- Contains the ``mongod`` server, default configuration files,
- and init.d scripts.
-
-:``Percona-Server-MongoDB-36-shell``:
- Contains the ``mongo`` shell.
-
-:``Percona-Server-MongoDB-36-mongos``:
- Contains the ``mongos`` sharded cluster query router.
+Installing from Percona Repositories
+================================================================================
 
-:``Percona-Server-MongoDB-36-tools``:
- Contains Mongo tools for high-performance MongoDB fork from Percona.
+It is recommended to intall |PSMDB| from official Percona repositories:
 
-:``Percona-Server-MongoDB-36-debuginfo``:
- Contains debug symbols for the server.
+|tip.run-all.root|
 
-Installing from Percona Repositories
-====================================
+1. Install the Percona repository: :bash:`yum install https://repo.percona.com/centos/7/RPMS/noarch/percona-release-0.1-8.noarch.rpm`
 
-It is recommended to intall |PSMDB| from official Percona repositories:
+   .. admonition:: Example of Output
 
-1. Configure Percona repositories as described in
-   `Percona Software Repositories Documentation
-   <https://www.percona.com/doc/percona-repo-config/index.html>`_.
+      .. code-block:: bash
 
-#. Install the required |PSMDB| package using :command:`yum`.
-   For example, to install the full package, run the following:
+	 Retrieving http://www.percona.com/downloads/percona-release/redhat/0.1-6/percona-release-0.1-8.noarch.rpm
```

Comment:
This link is wrong, please check with Alexey Bychko which one should be used and if possible use "latest" instead of the one with specific version.